### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by caitlinmallen** [thedunston/sshwireguard#6]
The documentation can be a bit hard to follow and I did see there was a header 3 markdown error where the markdown does not turn "###Wireguard Setup" into an actual header. 
I'd recommend making it a bit easier to follow since it feels a bit scattered. One way to do this could possibly be making a new section entirely for manual set-up instructions and also shortening or breaking down the larger paragraphs. When I read instructions I don't like having to read larger paragraphs on GitHub
![WireguardMD](https://user-images.githubusercontent.com/47792476/112773855-a3117a80-9005-11eb-94cf-8065cf8d0386.png)

**Comment by caitlinmallen** [comment]
Also specifically, installing on Windows 10 is hard to navigate. I'd recommend making a section explicitly for Windows 10 install as a client

**Comment by thedunston** [comment]
Windows 10 only requires the Powershell script.  Please clarify what install aspects are unclear.

**Comment by thedunston** [comment]
The paragraph lengths will stay as is so folks know what is happening behind the scenes.  However, I will add sections to separate out ideas.  Thank you for pointing out the header Markdown issue.

**Comment by thedunston** [comment]
The README has been updated and the documentation added to the wiki and separated out.

**Comment by caitlinmallen** [comment]
Cool! I actually forked the repo and was going to make it a bit clearer for Windows, I did have an issue with the Powershell script since I was not exactly sure how to really use the script at first and ended up asking Brandon Wilbur for guidance. I did also have an issue since my wgvpn.conf file was not working properly since it did not seems to be populating the information and I ended up actually copying the one that worked on a Linux host. I was going to add how to properly fill in and execute the Powershell script in the correct order as well as what to do if anybody runs into the same issue as me.

**Comment by caitlinmallen** [comment]
I did however get a Minecraft server to work with NoWire so I was able to eventually figure it out! Could also just be a me thing since nothing ever works the first time with me! https://github.com/caitlinmallen/sec440/blob/main/VPNLab/Readme.md

**Comment by thedunston** [comment]
Great.  Feel free to reach out to me for clarifications or suggestions too.  The client-scripts will be updated later this week to allow editing those files with the IP to ping and the VPN server to include in the client scripts.

**Comment by thedunston** [comment]
That is really cool!!
